src/commands/test/lib/nunit-xml-util.ts

```diff
@@ -62,6 +66,25 @@ export class NUnitXmlUtil extends XmlUtil {
     return xml1;
   }
 
+  combineNUnit3(xml1: Document, xml2: Document): Document {
+    this.combineTestResultsAttribute(xml1, xml2, "testcasecount");
+    this.combineTestResultsAttribute(xml1, xml2, "total");
+    this.combineTestResultsAttribute(xml1, xml2, "passed");
+    this.combineTestResultsAttribute(xml1, xml2, "failed");
+    this.combineTestResultsAttribute(xml1, xml2, "inconclusive");
+    this.combineTestResultsAttribute(xml1, xml2, "skipped");
+    this.combineTestResultsAttribute(xml1, xml2, "asserts");
+
+    const testSuitesParent: Element = this.collectAllElements(xml1.documentElement, "test-run")[0];
+    const testSuites: Element[] = this.collectChildren(xml2.documentElement, "test-suite");
+
+    testSuites.forEach((child: Element) => {
```

Comment:
Nice!

Reply:
I couldn't get this to work:

`.appendChild` isn't compatible with `...` because it expects at least 1 child element [this error gets raised](https://github.com/Microsoft/TypeScript/issues/28010)

`.append` appears to not work for an `Element` which is confusing. Unless you're super keen for me to do this I suggest we don't invest _too_ much time in it.